packages/react-conformance/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "@fluentui/react-conformance",
+  "version": "0.1.0",
+  "description": "Conformance testing",
+  "private": true,
+  "main": "lib/index.js",
+  "typings": "lib/index.d.ts",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/microsoft/fluentui"
+  },
+  "license": "MIT",
+  "scripts": {
+    "build": "just-scripts build",
+    "clean": "just-scripts clean",
+    "code-style": "just-scripts code-style",
+    "just": "just-scripts",
+    "lint": "just-scripts lint",
+    "start-test": "just-scripts jest-watch",
```

Comment:
I think for now you should remove `start-test`, `test`, and `update-snapshots` because right now the package contains tests for reuse but doesn't actually have any of its own tests. (Though later we should probably add some basic tests for the tests.)